apps/nova/src/pages/Root.tsx

```diff
@@ -46,14 +48,14 @@ const Root: React.FC = () => {
   return (
     <div
       className={cn("flex relative w-full", {
-        "ps-60": showNaviation,
+        "ps-60": showNaviation && lg,
```

Comment:
I believe moving `lg` to be part of `showNavigation` will be better. Remember to fix the typo in `showNaviation` please. 

```suggestion
        "ps-60": showNaviation && lg,
```